src/test/s3/commands/uploadFile.test.ts

```diff
@@ -11,6 +11,7 @@ import {
     getFilesToUpload,
```

Comment:
Would also be worth having test coverage for the use of globalState, similar to https://github.com/aws/aws-toolkit-vscode/blob/e9558c8776f41c152fd77f2f3de9839a1b19887c/src/test/s3/commands/downloadFileAs.test.ts#L60